### ISSUE TITLE
Include test for normality violation in t_test() and unianova()

### ISSUE THREAD

**Issue by LKobilke** [joon-e/tidycomm#49]
Since we provide feedback about the violation of normality in the regress() function, the idea came up that it might be beneficial to include this information in the `t-test()` and `unianova()` functions as well.

**Comment by MarHai** [comment]
Good idea. This would also allow us to streamline the code a bit -- and might thus be worth taking a look at before the christmas submit?
